source/tutorials/install.txt

```diff
@@ -0,0 +1,24 @@
+.. guide::
+   title: Installing MongoDB Build Tools
+   author: MongoDB Documentation Team
+   type: Deep Dive 
+   level: intermediate 
+   product_version: 4.0
+   result_description:
+     In this guide you will learn how to install the MongoDB Build Tools. Note that this guide only applies to MacOS officially.
```

Comment:
Suggest:
"In this guide you will learn how to install the MongoDB (should we say Documentation?) Build Tools on macOS."

- macOS is official capitalization
- I think we can combine both sentences and I'm wary to use words like "officially"

Reply:
done